drivers/i2s/Kconfig

```diff
@@ -35,6 +35,76 @@ config SYS_LOG_I2S_LEVEL
 	  3 INFO, write SYS_LOG_INF in addition to previous levels
 	  4 DEBUG, write SYS_LOG_DBG in addition to previous levels
 
+config	I2S_1
```

Comment:
The reason why I did it was because I saw it done in this way in SPI driver. I just prepared the new commit for the PR. I will test it out just a bit before pushing out.